website/thaliawebsite/settings.py

```diff
@@ -944,3 +952,14 @@ def from_env(
         "auth",
     ],
 }
+
+MONEYBIRD_ADMINISTRATION_ID = 380475297324075003
+MONEYBIRD_API_KEY = "-Xw8P1CfYFAA8Z0VEEnOgDvXiNNb8mgdBCgEMl05Yus"
+
+MONEYBIRD_AUTO_PUSH = True # Push changes to Moneybird automatically (so you don't have to call `instance.push_to_moneybird()` manually)
```

Comment:
I think these 2 settings, `MONEYBIRD_AUTO_PUSH` and `MONEYBIRD_FETCH_BEFORE_PUSH` are not used